Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM golang:alpine as builder
+FROM golang:1.17-alpine as builder
 
 # Install dependencies for copy
-RUN apk add -U --no-cache ca-certificates tzdata
+RUN apk add -U --no-cache ca-certificates tzdata git
 
 # Use an valid GOPATH and copy the files
 WORKDIR /go/src/github.com/cooperspencer/gickup
 COPY . .
 
 # Fetching dependencies and build the app
+RUN go mod tidy
```

Comment:
There `go mod tidy` is useless. It is useful to do it just after copying `go.mod` and go.sum` because, thanks to docker caching system, go will download dependencies only when a new dependency will be added.

It should be done like this : 
```Dockerfile
COPY go.mod .
COPY go.sum .
RUN go mod tidy # this is run only when go.mod or go.sum changed

COPY . .
```

Reply:
thanks, added it